Failing test in MDAnalysisTests.analysis.test_helanal

I ran the test suite on my local machine with the last version of the develop branch (commit 7b91b2a). The test `MDAnalysisTests.analysis.test_helanal.test_helanal_main` failed while it seems to pass on Travis.

The error message I get is:

```
======================================================================
FAIL: MDAnalysisTests.analysis.test_helanal.test_helanal_main
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jon/Envs/mda/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/jon/dev/mdanalysis/testsuite/MDAnalysisTests/analysis/test_helanal.py", line 132, in test_helanal_main
    err_msg="data[{0}] mismatch".format(label))
  File "/home/jon/Envs/mda/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 886, in assert_array_almost_equal
    precision=decimal)
  File "/home/jon/Envs/mda/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 708, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 5 decimals
data[Local bending angles] mismatch
(mismatch 23.8095238095%)
 x: array([  3.44526,   4.85426,   4.69544,   2.39474,   3.56172,   3.97128,
         3.41564,   1.8613 ,   5.22997,   5.41381,  27.49601,  39.69839,
        35.05922,  21.78929,   9.86322,   8.80069,   5.53445,   6.14354,
        10.15451,  11.07686,   9.23542], dtype=float32)
 y: array([  3.44526,   4.85426,   4.69548,   2.39474,   3.56172,   3.97128,
         3.41564,   1.86141,   5.22997,   5.41381,  27.49601,  39.69839,
        35.05922,  21.78929,   9.86324,   8.80067,   5.53446,   6.14357,
        10.15451,  11.07686,   9.23542], dtype=float32)
-------------------- >> begin captured logging << --------------------
MDAnalysis.core.AtomGroup: DEBUG: Universe.load_new(): loading /home/jon/dev/mdanalysis/testsuite/MDAnalysisTests/data/adk_open.pdb...
MDAnalysis.core.AtomGroup: DEBUG: Universe.load_new(): loading /home/jon/dev/mdanalysis/testsuite/MDAnalysisTests/data/adk_open.pdb...
MDAnalysis.analysis.helanal: INFO: Analysing 27/214 residues
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=87.8054 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=75.2341 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=121.771 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=134.948 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=58.9622 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=104.834 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=87.402 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=89.132 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=105.266 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=49.3685 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=54.2456 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=110.827 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=85.3605 degrees
MDAnalysis.analysis.helanal: DEBUG: Big Screw Up: screw_angle=68.2308 degrees
MDAnalysis.analysis.helanal: INFO: Height: 1.52861  SD: 0.196483  ABDEV: 0.113843  (Angstroem)
MDAnalysis.analysis.helanal: INFO: Twist: 98.8301  SD: 4.08172  ABDEV: 3.07253
MDAnalysis.analysis.helanal: INFO: Residues/turn: 3.64864  SD: 0.152694  ABDEV: 0.11314
MDAnalysis.analysis.helanal: INFO: Local bending angles:      3.4	      4.9	      4.7	      2.4	      3.6	      4.0	      3.4	      1.9	      5.2	      5.4	     27.5	     39.7	     35.1	     21.8	      9.9	      8.8	      5.5	      6.1	     10.2	     11.1	      9.2	
MDAnalysis.analysis.helanal: INFO: Unit twist angles:     97.2	     99.1	     97.3	    101.8	    100.4	     97.1	     97.1	     98.3	     97.9	     95.5	     97.1	     95.3	     87.9	    108.4	     95.3	    104.0	    100.7	    101.5	     99.6	     94.8	    102.5	     97.3	    104.5	    101.4	
MDAnalysis.analysis.helanal: INFO: Best fit tilt: 1.33097
MDAnalysis.analysis.helanal: INFO: Rotation Angles from 1 to n-1 (local screw angles): 87.8 -171.9 -75.2 24.6 121.8 -134.9 -35.1 59.0 159.4 -104.8 -7.5 87.4 -176.1 -89.1 17.2 105.3 -147.0 -49.4 54.2 156.1 -110.8 -5.7 85.4 -167.3 -68.2
--------------------- >> end captured logging << ---------------------
```

I run the tests with `./mda_nosetests --with-memleak --processes=-1 --process-timeout=300 analysis/test_helanal.py`.


From a bisect, the first commit that make the tests fails is e005d95b6ab9d0af106edb1fa2f178e0856053a1

--- a/testsuite/MDAnalysisTests/analysis/test_helanal.py
+++ b/testsuite/MDAnalysisTests/analysis/test_helanal.py
@@ -128,7 +128,7 @@ def test_helanal_main(reference=HELANAL_SINGLE_DATA):
     assert_equal(sorted(data.keys()), sorted(ref.keys()),
                      err_msg="different contents in data dict")
     for label in ref.keys():
-        assert_array_almost_equal(data[label], ref[label], decimal=5,
+        assert_array_almost_equal(data[label], ref[label], decimal=4,
                                   err_msg="data[{0}] mismatch".format(label))
 
 def test_xtc_striding():

